src/omnicore/tx.cpp

```diff
@@ -82,6 +82,16 @@ bool CMPTransaction::isOverrun(const char* p)
 
 // -------------------- PACKET PARSING -----------------------
 
+/** Temporarily disables packet debugging and then calls interpret_Transaction(). */
+bool CMPTransaction::reinterpret_Transaction()
+{
+    bool packetDebugState = msc_debug_parser;
+    msc_debug_parser = 0;
```

Comment:
I think this could be useful for another aspect: if the RPC only flag is enabled (which could be by default), then this could also be used to reject the use/access of `interpretPacket()`, basically ensuring no logic is "accidently" executed, outside of the normal handler flow.

All this seems a bit like a workaround though.